datadog/tests/resource_datadog_dashboard_scatterplot_test.go

```diff
@@ -101,6 +101,70 @@ resource "datadog_dashboard" "scatterplot_dashboard" {
 }
 `
 
+const datadogDashboardScatterplotFormulaConfig = `
+resource "datadog_dashboard" "scatterplot_dashboard" {
+	title         = "{{uniq}}"
+	description   = "Created using the Datadog provider in Terraform"
+	layout_type   = "ordered"
+	is_read_only  = "true"
+
+	widget {
+		scatterplot_definition {
+			request {
+				scatterplot_table {
+				  formula {
+					formula_expression = "my_query_1"
+					alias              = "first_query"
+					dimension          = "x"
+				  }
+				  formula {
+					formula_expression = "my_query_2"
+					alias              = "second_query"
```

Comment:
Let's do one without alias, to check the optionality 🙂

Reply:
Done in [4adda78](https://github.com/DataDog/terraform-provider-datadog/pull/1275/commits/4adda78b4a8ca09fae602014dce83c25f4e807eb).